### PR TITLE
Show that LDAP users cannot be deactivated

### PR DESCRIPTION
**Pull request by DHaussermann**
#### Is the feature in stable branch or under review?
- [] Feature in stable branch (master, main)?
- [x] Feature still under review? If yes, list all related pull requests for reference.
https://github.com/mattermost/mattermost/pull/25167

#### Test client

- [x] Browser
- [] Mobile App (version, OS)
- [x] Desktop App (version, OS)

### CODE REVIEW

**Review by saturninoabril — APPROVED** [review]
LGTM and thanks @DHaussermann!

File: data/test-cases/suite/ad-ldap/sysadmin-ui-shows-ldap-users-do-not-deactivate.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+# (Required) Ensure all values are filled up
+name: "System cossole user list shows that deactivate option is not available for LDAP users"
```
Comment (lindalumitchell) [inline data/test-cases/suite/ad-ldap/sysadmin-ui-shows-ldap-users-do-not-deactivate.md:3]:
```suggestion
name: "System Console user list shows that deactivate option is not available for LDAP users"
```

**Review by lindalumitchell — APPROVED** [review]
Sorry for the delay, @DHaussermann! Looks good. Made one teensy suggestion. 👍

**Comment by saturninoabril** [comment]
Test case saved as MM-T5517

**Pull request merged**